pkg/manifest/build.go

```diff
@@ -152,6 +159,13 @@ func (p *BuildrootFromPackages) serialize() osbuild.Pipeline {
 	return pipeline
 }
 
+func (p *BuildrootFromPackages) getSELinuxPolicy() string {
+	if p.selinuxPolicy != "" {
```

Comment:
Are you using fully custom SELinux policy packages or could this validate against the set of 'usual suspects'?